test/waves/photoelectric_effect.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
```

Comment:
Test's name should end with `test`

Reply:
Changed filename to `photoelectron_energy_from_frequency_test.py`